src/components/base/outline-accordion/outline-accordion.stories.ts

```diff
@@ -12,35 +12,35 @@ export default {
         code: `
   <outline-accordion label=label clean=clean single-panel=singlePanel>
     <outline-accordion-panel slot="panels">
-      <h5 slot="heading">Heading 1</h5>
+      <span slot="heading">Heading 1</span>
       <div class="wysiwyg">
```

Comment:
Let's replace the usage of `<div class="wysiwyg">` with `<outline-styled-text>` This is the new wrapper for any content supplied by the editor that should include basic styling for generic HTML elements.